R/chartutils.R

```diff
@@ -102,17 +103,24 @@ checkMatrixNames <- function(x, assign.col.names = TRUE)
         dimnames(new.x) <- old.names
     }
 
-    # Many of the charts use the statistic attribute
-    x <- CopyAttributes(new.x, x)
+    # Convert percentage data to decimal form
+    stat <- attr(x, "statistic")
```

Comment:
Why is this line repeated (line 93)

Reply:
Thanks for picking that up. I've removed the repeated line in 47c0894